app/models/chart_data_values.rb

```diff
@@ -183,13 +183,29 @@ def self.as_chart_json(output)
       :y1_axis_choices,
       :explore_message,
       :pinch_and_zoom_message,
-      :click_and_drag_message
+      :click_and_drag_message,
+      :subtitle_start_date,
+      :subtitle_end_date
     ].inject({}) do |json, field|
       json[field] = output.public_send(field)
       json
     end
   end
 
+  def subtitle_start_date
+    return nil unless x_axis_ranges_present?
+    transformations_empty_or_only_move? ? format_subtitle_date(@x_axis_ranges.first.first) : nil
+  end
```

Comment:
perhaps this could be reworked so both `nil` returns are included in the early return?  (similar for `subtitle_end_date` below?)
```
  def subtitle_start_date
    return nil unless x_axis_ranges_present? && transformations_empty_or_only_move?
    
    format_subtitle_date(@x_axis_ranges.first.first)
  end
```